### ISSUE TITLE
I have a question about your ESANet model inference time.

### ISSUE THREAD

**Issue by suesuekkim** [TUI-NICR/ESANet#35]
Firstly, I really appreciate about your clean and kind code.

I have trained ESANet model with SUNRGBD dataset and scenenet dataset. For the SUNRGBD dataset, when i implemented the code prepare_dataset.py in the src colder, the refined depth data was generated, and when i train the depth mode to 'refined', it was trained with the refined depth dataset. And I have a question here, in the paper of this model, is the inference time that you mentioned in Fig 4. and Fig 5. include the data pre-processing time (making the raw depth to refined depth)? 

Thank you for reading my issue.

**Comment by mona0809** [comment]
Yes, we did train on refined depth data as it common practice. Still, you can use the model and apply it on "raw depth" samples. However, you can also train on raw depth data, which results in only slightly lower mIoU. For application, I think, you will not notice the difference.
